R/estimate_setting_contacts.R

```diff
@@ -8,13 +8,15 @@
 #' @param contact_data_list PARAM_DESCRIPTION
 #' @param survey_population PARAM_DESCRIPTION
 #' @param prediction_population PARAM_DESCRIPTION, Default: survey_population
-#' @param age_breaks PARAM_DESCRIPTION
+#' @param age_breaks PARAM_DESCRIPTION 
+#' @param per_capita_household_size PARAM_DESCRIPTION, Default : NULL
 #' @return OUTPUT_DESCRIPTION
```

Comment:
Update these arguments with brief examples. For example,

```
#' @param age_breaks vector of age values, e.g., c(0,4,5,9,10,14)
```

Note for the age_breaks - are we expecting character, like ("0-4") or numeric?

Also will be worth briefly mentioning why the default for `per_capita_household_size` is NULL